client/extensions/woocommerce/state/sites/promotions/reducer.js

```diff
@@ -98,6 +98,7 @@ function calculatePromotions( coupons, products ) {
 
 function createPromotionFromProduct( product ) {
 	return {
+		id: 'product:' + product.id,
```

Comment:
This will limit us to creating one promotion from a product - is that going to be ok?

Reply:
It's a technical limitation of a product sale.